mixer/adapter/fluentd/config/config.proto

```diff
@@ -40,4 +40,9 @@ message Params {
   // Address of listening fluentd daemon. Example: fluentd-server:24224
   // Default value is localhost:24224
   string address = 1;
+
+  // Convert attributes of type duration to integers of unit ms in
+  // logs. Default behaviour is a string representation including
+  // unit.
+  bool integer_duration = 2;
```

Comment:
I'm not sure if this should be an enum instead.

Reply:
enum would be more flexible, but I'm not quite sure what the other values would be.

should we have integer_duration be the default (and name this something like: `durations_as_string`) ?

Reply:
I don't mind switching the default, if that makes the most sense.